release_util/management/commands/check_reserved_keywords.py

```diff
@@ -22,6 +26,9 @@
 * --override_file: pass a yaml file containing a list of model fields to ignore when scanning
   for reserved keyword violations.
 * --report_path: pass a path in which to write a csv report of the violations found with this script.
+* --report_file: name of file in which to write violation report data
+* --system: an individual system, defined in `reserved_keyword_file`, for which you will scan for
```

Comment:
Nit: This confused me a little, even knowing what's going on here. Maybe something like: "The type of database management system (mysql, snowflake, ...), as defined in ..."?

Reply:
yeah, I sat for a while trying to figure out a way to make this make sense. Will fix